qa/suites/rbd/cache/workloads/fio.yaml

```diff
@@ -0,0 +1,32 @@
+tasks:
+- install:
+    extra_packages: ['ceph-immutable-object-cache']
+- ceph:
+    conf:
+      osd:
+        osd shutdown pgref assert: true
+      client:
+        rbd_parent_cache_enabled: true
+        #debug immutable_obj_cache: 20/20
+        immutable object cache path: /tmp/ceph-immutable-object-cache
+        immutable object cache max size: 50G
```

Comment:
Hmm -- I'm not sure any teuthology node has this amount of space just sitting around idle.